### PR TITLE
Added documentation for the Finder::reverseSorting() method

### LINKED ISSUE #10577: [Finder] Added a way to inverse a previous sorting
https://github.com/symfony/symfony/pull/27967

### PR DESCRIPTION
**Pull request by TimoBakx**
Fixed #10577.

### CODE REVIEW

File: components/finder.rst [hunk 1]
```diff
@@ -192,6 +192,15 @@ You can also define your own sorting algorithm with ``sort()`` method::
...
     });
 
+You can reverse any sorting by using the ``reverseSorting()`` method::
+
+    $finder->sortByName()->reverseSorting();
+
+    $finder->sortByModifiedTime()->reverseSorting();
+
+.. versionadded:: 4.2
+    The option to reverse the sort order was introduced in Symfony 4.2.
```
Comment (OskarStark) [inline components/finder.rst]: The „reverseSorting()“ method to reverse the sort order was.......

File: components/finder.rst [hunk 2]
```diff
@@ -192,6 +192,15 @@ You can also define your own sorting algorithm with ``sort()`` method::
         return strcmp($a->getRealPath(), $b->getRealPath());
     });
 
+You can reverse any sorting by using the ``reverseSorting()`` method::
+
+    $finder->sortByName()->reverseSorting();
+
+    $finder->sortByModifiedTime()->reverseSorting();
```
Comment (OskarStark) [inline components/finder.rst]: Do we Need 2 examples here?

Comment (TimoBakx) [reply components/finder.rst]: Good point. Removed the latter.

File: components/finder.rst (hunk 1 above)
Comment (TimoBakx) [reply components/finder.rst]: Updated.

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Timo for contributing this! And congrats on your first Symfony Docs contribution! 🎉

**Comment by TimoBakx** [comment]
Thank you @javiereguiluz for merging this. And thanks @OskarStark for your review.

**Comment by OskarStark** [comment]
You‘re welcome 🤗

**Pull request merged**